www_src/components/select/select.less

```diff
@@ -0,0 +1,12 @@
+.select {
+  -webkit-appearance: none;
```

Comment:
small bump - if we can append to the list of "to prefix" elements, that sounds like worth doing here.